shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -273,13 +288,28 @@ export default {
       isAuthenticated:                          this.provider !== GOOGLE || this.mode === _EDIT,
       projectId:                                null,
       REGISTRIES_TAB_NAME,
-      labelForAddon
-
+      labelForAddon,
+      s3ConfigValue:                            { ...initialS3Config },
     };
   },
 
   computed: {
     ...mapGetters({ features: 'features/get' }),
+
+    isS3EndpointTrulyValid() {
```

Comment:
See https://github.com/rancher/dashboard/pull/14557#discussion_r2168577814

This still does S3Config specific validation outside of the S3Config component